packages/front-end/components/Features/FeatureVariationsInput.tsx

```diff
@@ -90,20 +92,20 @@ export default function FeatureVariationsInput({
             </label>
             <div className="row align-items-center pb-3">
               <div className="col">
-                <input
-                  value={isNaN(coverage) ? 0 : decimalToPercent(coverage)}
-                  onChange={(e) => {
-                    let decimal = percentToDecimal(e.target.value);
+                <Slider
+                  name="RadixSlider"
+                  defaultValue={[50]}
```

Comment:
Specify either `defaultValue` or `value`, but  not both.  In this case, we want to use `value` since it's a controlled component.  Also, you do not need to specify a `name`.

Same comment for the other file.

Reply:
Sounds good!